autoware_launch/rviz/autoware.rviz

```diff
@@ -61,22 +61,6 @@ Visualization Manager:
           Value: false
         - Class: rviz_common/Group
           Displays:
-            - Class: rviz_plugins/SteeringAngle
-              Enabled: true
```

Comment:
I think you can disable this by just making `Enabled` false.

Reply:
good point. I reverted the change and only changed the `Enabled` value